src/app/main-dashboard/main-dashboard.component.html

```diff
@@ -3,7 +3,8 @@
   <aside class="col-sm-3 pl-0">
     <app-sidebar></app-sidebar>
   </aside>
-  <main class="col-sm-9 p-0 bg-light-gery">
+
+  <main class="col-sm-9 p-0 bg-light-gery ">
```

Comment:
@shreyas1434shinde fix typo should be grey

Reply:
@itsvick done